.github/workflows/ci.yml

```diff
@@ -376,7 +378,8 @@ jobs:
 
       # This will be a no-op under normal circumstances since the cluster would have been deployed
       # in deploy-cluster. It is needed in case we want to re-run the job after the cluster has been reaped.
-      - uses: realm/ci-actions/mdb-realm/deployApps@3f810b2d04e9dada2bde0b33ec90102e52a0b30a
+      - name: Try to create cluster
```

Comment:
```suggestion
      - name: Create cluster
```

Reply:
@desistefanova is this resolved. Why not name it Create cluster as the flutter-ios does. Just for consistency

Reply:
Oh I may be missing something. There are multiple of these resolved. Seems you have discussed this already. Feel free to ignore if that is the case